dhis-2/dhis-services/dhis-service-setting/src/main/java/org/hisp/dhis/setting/DefaultSystemSettingManager.java

```diff
@@ -214,7 +210,7 @@ public Serializable getSystemSetting( SettingKey key, Serializable defaultValue
             {
                 try
                 {
-                    return Optional.of( pbeStringEncryptor.decrypt( (String) setting.getValue() ) );

```

Comment:
This comment is not related to your change, but I would consider removing the `TransactionTemplate` in this method and use the `@Transactional(readOnly)` annotation, since it's used in other methods of the same class

Reply:
Fixed